rst_files/lln_clt.rst

```diff
@@ -602,47 +630,54 @@ In the figure, the closest density is that of :math:`Y_1`, while the furthest is
       return (X .- m) ./ sigma
   end
 
-  let
-      nmax = 5
-      reps = 100000
-      ns = 1:nmax
-
-      # == Form a matrix Z such that each column is reps independent draws of X == #
-      Z = zeros(reps, nmax)
-      for i ∈ ns
-          Z[:, i] = gen_x_draws(reps)
-      end
-
-      # == Take cumulative sum across columns
-      S = cumsum(Z, dims = 2)
-
-      # == Multiply j-th column by sqrt j == #
-      Y = S .* (1. ./ sqrt.(ns))'
-
-      # == Plot == #
-      a, b = -3, 3
-      gs = 100
-      xs = range(a, stop = b, length = gs)
-
-      x_vec = []
-      y_vec = []
-      z_vec = []
-      colors = []
-      for n ∈ ns
-          kde_est = kde(Y[:, n])
-          _xs, ys = kde_est.x, kde_est.density
-          push!(x_vec, collect(_xs))
-          push!(y_vec, ys)
-          push!(z_vec, collect(n*ones( length(_xs))))
-          push!(colors, RGBA(0, 0, 0, 1-(n-1)/nmax))
-      end
-
-      plot(x_vec, z_vec, y_vec, color = reshape(colors,1,length(colors)),
-           legend = :none)
-      plot!(xlims = (a,b), xticks = [-3; 0; 3], ylims = (1, nmax), yticks = ns,
-            ylabel = "n", xlabel = "\$ Y_n \$", zlabel = "\$ p(y_n) \$",
-            zlims=(0, 0.4), zticks=[0.2; 0.4])
-  end
+    nmax = 5
+    reps = 100000
+    ns = 1:nmax
+
+    # == Form a matrix Z such that each column is reps independent draws of X == #
+    Z = zeros(reps, nmax)
+    for i ∈ ns
+        Z[:, i] = gen_x_draws(reps)
+    end
+
+    # == Take cumulative sum across columns
+    S = cumsum(Z, dims = 2)
+
+    # == Multiply j-th column by sqrt j == #
+    Y = S .* (1. ./ sqrt.(ns))'
+
+    # == Plot == #
+    a, b = -3, 3
+    gs = 100
+    xs = range(a, stop = b, length = gs)
+
+    x_vec = []
+    y_vec = []
+    z_vec = []
+    colors = []
+    for n ∈ ns
+        kde_est = kde(Y[:, n])
+        _xs, ys = kde_est.x, kde_est.density
+        push!(x_vec, collect(_xs))
+        push!(y_vec, ys)
+        push!(z_vec, collect(n*ones( length(_xs))))
+        push!(colors, RGBA(0, 0, 0, 1-(n-1)/nmax))
```

Comment:
is it my screen or does the last `push!` looks italic?

Reply:
Looks italic for me too. This is probably happening because of the `_` in the variable `_xs`. Seems like a strictly Markdown/GitHub thing.